client/components/status/status.js

```
@@ -903,14 +957,18 @@ export default class Status extends React.Component {
+    
     await this.getUserPastRadiusSessions();
+    if (!this.isComponentMounted) return;
     await this.getUserActiveRadiusSessions();
+    if (!this.isComponentMounted) return;
     setLoading(false);
   }
 
   async fetchMoreSessions() {
     const {currentPage} = this.state;
     await this.getUserPastRadiusSessions({page: currentPage + 1});
+    if (!this.isComponentMounted) return;
```

ESLint

[error] Unnecessary return statement.

(no-useless-return)